staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -1579,6 +1579,9 @@ func TestCheckRetryClosesBody(t *testing.T) {
 		count++
 		t.Logf("attempt %d", count)
 		if count >= 5 {
+			if data, err := ioutil.ReadAll(req.Body); err != nil || !bytes.Equal(data, []byte(strings.Repeat("abcd", 1000))) {
```

Comment:
Stupid question, but I read this and get confused whether the read all on the server side has a side effect for the client code?  Maybe it's because it's monday...

Reply:
no, this just verified on the server side that the body actually got reset, so that the data actually sent with the last accepted request was the full body